src/status_im2/navigation/screens.cljs

```diff
@@ -33,7 +33,24 @@
     [status-im2.contexts.share.view :as share]
     [status-im2.contexts.onboarding.syncing.results.view :as syncing-results]
     [status-im2.contexts.onboarding.syncing.progress.view :as syncing-devices]
-    [status-im2.contexts.chat.new-chat.view :as new-chat]))
+    [status-im2.contexts.chat.new-chat.view :as new-chat]
+    [react-native.core :as rn]
+    [quo.platform :as platform]
+    [status-im2.constants :as constants]))
+
+(def sign-in-modal-animations
+  {:showModal    {:translationY {:from     (- (:height (rn/get-window)) (if platform/ios? 318 296))
```

Comment:
hmm, where are we getting these values from? perhaps we can use common values or at least named vars for what `318` and `296` are in relation to

Reply:
Removed it as no longer needed